ts/store/middlewares/analytics.ts

```diff
@@ -227,8 +227,8 @@ const trackAction = (mp: NonNullable<typeof mixpanel>) => (
     case getType(identificationPinReset):
     // onboarding
     case getType(analyticsOnboardingStarted):
-    case getType(tosAccept.success):
     case getType(createPinSuccess):
+    case getType(tosAccepted):
```

Comment:
I don't know if you are working on it right now and it will be done in next commit, anyway you should forward to mixpanel the tos version accepted by the user https://github.com/teamdigitale/io-app/pull/1047#discussion_r316551579

Reply:
ye, I added it within the last commit. Don't know  why it is not displayed here on github.com. I see it on git logs:
<img width="956" alt="image" src="https://user-images.githubusercontent.com/38431762/63504035-f427c700-c4d0-11e9-9ad4-7db7b1f2e5f5.png">

Here the changes:
<img width="1309" alt="image" src="https://user-images.githubusercontent.com/38431762/63504101-1588b300-c4d1-11e9-8dbe-b73969841db3.png">

<img width="1312" alt="image" src="https://user-images.githubusercontent.com/38431762/63504122-20dbde80-c4d1-11e9-8bfa-7974be1dff22.png">

Reply:
@valentinacala with the last commit I can see the changes 👍